app/javascript/components/assessment/steps/SymptomsAssessment.js

```diff
@@ -144,13 +144,21 @@ class SymptomsAssessment extends React.Component {
     const id = `${symp.name}${this.props.idPre ? '_idpre' + this.props.idPre : ''}`;
     return (
       <Form.Row className="pt-3" key={key}>
-        <Form.Label className="nav-input-label" key={key} htmlFor={id}>
+        <Form.Label className="nav-input-label" key={key + '_label'} htmlFor={id}>
```

Comment:
Can you describe what this change is doing? Should `htmlFor={id}` be changed to `htmlFor={id + '_control'}` as well?

Reply:
We were using the same key in multiple places and I was getting the warning about children with duplicate keys in the console while debugging. This changes the key slightly so they are not all the same. 

As for the second pert of the question, I didn't change that one as it was the first place id was used but I can if we want to.

Reply:
I don't think we need to change the id/htmlFor here

Reply:
The label htmlFor and control id must match

Reply:
yes but the keys need to be different. just the line I commented on below needs to be changed